.dockerignore

```diff
@@ -1,2 +1 @@
-.git
```

Comment:
Why are you removing this? Using `make docker-compose-build` runs

https://github.com/ansible/awx/blob/642e6f792cad6365506d2ae1f48f2a828b290fb8/Makefile#L583-L584

And this passes `.` for the build context. I agree it would be better to use another folder for the build context, but I don't see anything in this PR that makes that change. Without that, we shouldn't be including folders like this.

Reply:
One of the things I've done in this PR is "Removes need for awx_sdist_builder image". To do that, We are now building the sdist in the first stage of the image build. A UI target currently calls git. /cc @jakemcdermott is this something we can fix / change?

https://github.com/ansible/awx/blob/642e6f792cad6365506d2ae1f48f2a828b290fb8/installer/roles/image_build/templates/Dockerfile.j2#L79-L82

https://github.com/ansible/awx/blob/642e6f792cad6365506d2ae1f48f2a828b290fb8/Makefile#L484

Reply:
that was such an expected answer! Yeah, we could refactor that but I'm also fine leaving it temporarily if it still builds fast enough.